src/components/atoms/InputText/__snapshots__/InputText.test.tsx.snap

```diff
@@ -22,7 +22,6 @@ exports[`<InputText /> renders the component disabled 1`] = `
   -webkit-transition: 0.2s ease-in-out;
   transition: 0.2s ease-in-out;
   font-family: "Open Sans",Roboto,Helvetica,Arial,sans-serif;
-  font-size: 16px;
```

Comment:
We should default to `font-size: 16px` so we don't break any consumers of the package

Reply:
this bit was automatically deleted when i ran -u to udpate the tests - do you want me to change it back?

Reply:
in InputText the default is set to 'body' which is equal to 16px, so no change to the default in reality